settings-api.lp

```diff
@@ -260,6 +260,9 @@ mg.include('scripts/pi-hole/lua/settings_header.lp','r')
                 <p>IMPORTANT: The app password generated here will only be shown
                 once and cannot be recovered. Make sure to store it in a safe
                 place!</p>
+                <div class="danger-area" style="display: none;" id="existing_apppw_warning">
```

Comment:
As an alternative to yubi's suggestion, we could use a different style for the warning box, using a different CSS classes:

```suggestion
                <div class="callout callout-warning" style="display: none;" id="existing_apppw_warning">
```

---

We also need to remove lines 25 to 27 from `default-dark.css` theme _(or change its color to `#fff`)_:
https://github.com/pi-hole/web/blob/3681d7187a44b32a667f40b585629d45bf4e44e9/style/themes/default-dark.css#L25-L27

![image](https://github.com/pi-hole/web/assets/1385443/03c80300-0336-4ea4-b6be-158e9a0d9812)